views/index.pug

```diff
@@ -18,11 +18,19 @@
     script(src="https://www.gstatic.com/firebasejs/7.14.3/firebase-app.js")
     script(src="https://www.gstatic.com/firebasejs/7.14.3/firebase-analytics.js")
     script(src="https://www.gstatic.com/firebasejs/7.14.3/firebase-auth.js")
-    script(src="/static/js/hj.js")
+    script(src="/static/js/third/hj.js")
     link(rel="apple-touch-icon" href="/static/imgs/appicon.png")
     link(rel="shortcut icon" type="image/jpg" href="/static/imgs/appicon.ico")
     link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.css')
     script(src="https://kit.fontawesome.com/89be96593a.js" crossorigin="anonymous")
+    link(rel="preconnect" href="https://in.hotjar.com")
+    link(rel="preconnect" href="https://static.hotjar.com")
+    link(rel="preconnect" href="https://vars.hotjar.com")
+    link(rel="preconnect" href="https://script.hotjar.com")
+    link(rel="preconnect" href="https://firebaseinstallations.googleapis.com")
+    link(rel="dns-prefetch" href="https://www.googletagmanager.com")
+    link(rel="dns-prefetch" href="https://kit-free.fontawesome.com")
+    <script id="mcjs">!function(c,h,i,m,p){m=c.createElement(h),p=c.getElementsByTagName(h)[0],m.async=1,m.src=i,p.parentNode.insertBefore(m,p)}(document,"script","https://chimpstatic.com/mcjs-connected/js/users/e7fc6811b410d9e7d8d5704ad/4ca2fe433d58314a32e654522.js");</script>
```

Comment:
let's talk about our approach to adding in pre-generated code (like iframe embeds or this). Do we want to adapt them to pug syntax or leave them as vanilla HTML?